### PR TITLE
Add Post method Export Instructions

### PR DESCRIPTION
**Pull request by jiwom**

### CODE REVIEW

File: buttons/export.md [hunk 1]
```diff
@@ -105,6 +105,33 @@ class UsersDataTable extends DataTable
 ...
 ```
 
+<a name="post-pdf"></a>
```
Comment (yajra) [inline buttons/export.md]: Maybe name it as `post-export`?

**Comment by jiwom** [comment]
Thanks! Done!

File: buttons/export.md [hunk 2]
```diff
@@ -132,6 +132,12 @@ class UsersDataTable extends DataTable
 ...
 ```
 
+And also add this code to your routes.php file.
+```php
+    $router->post('sample/export', SampleController::class . '@index');
```
Comment (yajra) [inline buttons/export.md]: Maybe also include the route for the main dataTable index for clarity?
```php
Route::resource('sample', 'SampleController@index');
Route::post('sample/export', 'SampleController@index');
```

Comment (jiwom) [reply buttons/export.md]: Done changing routes instruction.

**Pull request merged**